src/utils/getFilterdPeople.ts

```diff
@@ -0,0 +1,38 @@
+import { Person } from '../types';
+
+const getCentury = (year: number) => {
+  return Math.ceil(year / 100);
+};
+
+export function getFilterdPeople(
+  people: Person[],
+  sex: string,
+  query: string,
+  centuries: string[],
+) {
+  let filteredPeople = [...people];
```

Comment:
what is the reason to use let, you can change array even if it is constant

Reply:
![image](https://github.com/mate-academy/react_people-table-advanced/assets/139622699/51100443-b894-4a96-b75c-c33c2347f502)